nano/qt_test/qt.cpp

```diff
@@ -60,6 +62,38 @@ TEST (wallet, status)
 	ASSERT_TRUE (wallet_has (nano_qt::status_types::disconnected));
 }
 
+// this test is modelled on wallet.status but it introduces another node on the network
+TEST (wallet, status_with_peer)
+{
+	nano_qt::eventloop_processor processor;
+	nano::system system (2);
+	auto wallet_l (system.nodes[0]->wallets.create (nano::random_wallet_id ()));
```

Comment:
For a new test, we should use assignment initialization:

auto wallet_l = system.nodes[0]->wallets.create (nano::random_wallet_id ());